project/MimaExcludes.scala

```diff
@@ -79,6 +79,12 @@ object MimaExcludes {
     ProblemFilters.exclude[DirectMissingMethodProblem]("org.apache.spark.scheduler.SparkListenerApplicationStart.this"),
     ProblemFilters.exclude[MissingTypesProblem]("org.apache.spark.scheduler.SparkListenerApplicationStart$"),
     
+    // [SPARK-27630][CORE] Properly handle task end events from completed stages
```

Comment:
Please do not forget to update this comment once you update the PR title.

Reply:
I have compared PR title and comment to be consistent.
What comment are missing?

Reply:
Oops, well enough.